frontend/src/environments/environment.webnode-local.ts

```diff
@@ -1,5 +1,36 @@
 import { MinaEnv } from '@shared/types/core/environment/mina-env.type';
 
+/** Substituted on launch of docker container from MINA_WEBNODE_SEED_URLS by `ng build` */
+declare const MINA_WEBNODE_SEED_URLS: string;
+
+/** Default list of seed URLs if MINA_WEBNODE_SEED_URLS isn't specified */
+const defaultWebNodeSeedUrls: Readonly<string[]> = [
+  'https://bootnodes.minaprotocol.com/networks/devnet-webrtc.txt',
+  '/webnode/pkg/devnet-webrtc.txt',
+];
+
+/** Substituted on launch of docker container from MINA_WEBNODE_BOOTNODES by `ng build` */
+declare const MINA_WEBNODE_BOOTNODES: string;
+
+/** Default list of bootnodes if MINA_WEBNODE_BOOTNODES is unspecified */
+const defaultWebNodeBootNodes: Readonly<string[]> = [
+  // example:
+  // "/2az589QvS6i3EJiVKUfVHCkyqf4khGy9PjQF7nSQuveF27wp7xX/https/mina-rust-seed-1.gcp.o1test.net/443",
+];
+
+function commaSeparatedEnv(
+  envVal: string,
+  fallback: Readonly<string[]>,
```

Comment:
Q: it is only about wording, but why do you use the term `fallback` instead of `default`?

Reply:
Ah, fun.